src/XenobladeChroniclesX/Mods/CharacterSize/rules.txt

```diff
@@ -0,0 +1,445 @@
+category = "Height:"
+name = Normal
+condition = $characterID == 5
+Default = 1
+$height = 0.99 #0x3F7D70A4
+
+[Preset]
+category = "Height:"
+name = Normal
+condition = $characterID == 6
+Default = 1
+$height = 0.97 #0x3F7851EC
+
+[Preset]
+category = "Height:"
+name = Normal
+condition = $characterID == 7
+Default = 1
+$height = 1.03 #0x3F83D70A
+
+[Preset]
+category = "Height:"
+name = Normal
+condition = $characterID == 8
+Default = 1
+$height = 1.06 #0x3F87AE14
+
+[Preset]
+category = "Height:"
+name = Normal
+condition = $characterID == 9
+Default = 1
+$height = 1.01 0x3F8147AE
+
+[Preset]
+category = "Height:"
+name = Normal
+condition = $characterID == 10
+Default = 1
+$height = 1.0 #0x3F800000
+
+[Preset]
+category = "Height:"
+name = Normal
+condition = $characterID == 11
+Default = 1
+$height = 1.02 #0x3F828F5C
+
+[Preset]
+category = "Height:"
+name = Normal
+condition = $characterID == 12
+Default = 1
+$height = 0.95 #0x3F733333
+
+[Preset]
+category = "Height:"
+name = Normal
+condition = $characterID == 13
+Default = 1
+$height = 0.85 #0x3F59999A
+
+[Preset]
+category = "Height:"
+name = Normal
+condition = $characterID == 14
+Default = 1
+$height = 0.87 #0x3F5EB852
+
+[Preset]
+category = "Height:"
+name = Normal
+condition = $characterID == 15
+Default = 1
+$height = 0.93 #0x3F6E147B
+
+[Preset]
+category = "Height:"
+name = Normal
+condition = $characterID == 16
+Default = 1
+$height = 0.95 #0x3F733333
+
+[Preset]
+category = "Height:"
+name = Normal
+condition = $characterID == 17
+Default = 1
+$height = 0.92 #0x3F6B851F
+
+[Preset]
+category = "Height:"
+name = Normal
+condition = $characterID == 18
+Default = 1
+$height = 0.97 #0x3F7851EC
+
+[Preset]
+category = "Height:"
+name = Normal
+condition = $characterID == 19
+Default = 1
+$height = 0.88 #0x3F6147AE
+
+[Preset]
+category = "Height:"
+name = Short (0.7)
+$height = 0.7
+
+[Preset]
+category = "Height:"
+name = Small (0.55)
+$height = 0.55
+
+[Preset]
+category = "Height:"
+name = Tiny (0.35)
+$height = 0.35
+
+[Preset]
+category = "Height:"
+name = Rat (0.15)
+$height = 0.15
+
+[Preset]
+category = "Height:"
+name = Ant (0.05)
+$height = 0.05
+
+[Preset]
+category = "Height:"
+name = Character Disapears (0)
+condition = $glitchheight == 1
+$height = 0
+
+[Preset]
+category = "Height:"
+name = Corrupt Hair [CRASHES the game after a bit] (-1)
+condition = $glitchheight == 1
+$height = -1
+
+[Preset]
+category = "Height:"
+name = Character Walks under ground [CRASHES the game after a bit] (-1.2)
+condition = $glitchheight == 1
+$height = -1.2
+
+[Preset]
+category = "Height:"
+name = Character Walks under ground and is Giant [CRASHES the game after a bit] (-10000)
+condition = $glitchheight == 1
+$height = -10000
+
+[Preset]
+category = "Height:"
+name = Character Walks under ground and is Massive [CRASHES the game after a bit] (-80000)
+condition = $glitchheight == 1
+$height = -80000
+
+[Preset]
+category = "Height:"
+name = Character Cant interact with terminals (NaN)
```

Comment:
The capitalization is a bit weird and its kinda long. Maybe mark it "Giant character walks underground [UNSTABLE] (-10000)"?

Reply:
I fixed this for all the negative options in the newest commit :+1: